README.md

```diff
@@ -68,13 +68,15 @@ The specification below describes the `trigger` metadata in `ScaledObject` resou
     - type: external
       metadata:
         scalerAddress: external-scaler-azure-cosmos-db.keda:4050 # Mandatory. Address of the external scaler service.
-        connection: <connection>               # Mandatory. Connection string of Cosmos DB account with monitored container.
-        databaseId: <database-id>              # Mandatory. ID of Cosmos DB database containing monitored container.
-        containerId: <container-id>            # Mandatory. ID of monitored container.
-        leaseConnection: <lease-connection>    # Mandatory. Connection string of Cosmos DB account with lease container.
-        leaseDatabaseId: <lease-database-id>   # Mandatory. ID of Cosmos DB database containing lease container.
-        leaseContainerId: <lease-container-id> # Mandatory. ID of lease container.
-        processorName: <processor-name>        # Mandatory. Name of change-feed processor used by listener application.
+        connection: <connection>                                 # Mandatory. Connection string of Cosmos DB account with monitored container.
```

Comment:
nit: now if someone provides connectionFromEnv, then `connection` is optional I guess? Should we change it from Mandatory to `optional` and update comment to say provide either this or that? same for leaseconnection